### PR TITLE
Fix some issues

### PR DESCRIPTION
**Pull request by seawatts**

### CODE REVIEW

File: config/initializers/objection-connect.ts [hunk 1]
```diff
@@ -7,6 +7,9 @@ export default {
   before: 'define-orm-models',
   async initialize(application: Application) {
     assert(application.config.database && application.config.database.client, 'Looks like you are missing database configuration. Add it to config.database - see the knex docs for configuration details: http://knexjs.org/#Installation-client');
-    application.container.register('objection:knex', Knex(application.config.database));
+    application.container.register('objection:knex', Knex(application.config.database), {
```
Comment (seawatts) [inline config/initializers/objection-connect.ts]: Not sure if this is actually needed.

File: app/orm-adapters/objection.ts [hunk 2]
```diff
@@ -189,10 +195,12 @@ export default class ObjectionAdapter extends ORMAdapter {
             return super.$parseDatabaseJson(json);
           }
         }
+
         Object.defineProperty(ObjectionModel, 'name', {
-          value: startCase(type) + 'ObjectionModel'
+          value: `${chain(type).startCase().replace(' ', '').value()}ObjectionModel`
```
Comment (davewasmer) [inline app/orm-adapters/objection.ts]: Using `chain()` here seems to just add verbosity, since we are only using a single lodash method.

File: app/orm-adapters/objection.ts [hunk 3]
```diff
@@ -34,9 +35,12 @@ declare module 'objection' {
 }
 
 export default class ObjectionAdapter extends ORMAdapter {
-
   objectionModels: { [type: string]: typeof ObjectionBaseModel };
-  knex = inject<knex>('objection:knex');
+  knex: knex;
+
+  init() {
+    this.knex = this.container.lookup('objection:knex');
```
Comment (davewasmer) [inline app/orm-adapters/objection.ts]: Why was this change needed?

File: config/initializers/objection-connect.ts (hunk 1 above)
Comment (davewasmer) [inline config/initializers/objection-connect.ts]: The default options are `singleton: true, instantiate: false` (https://github.com/denali-js/denali/blob/master/lib/metal/container.ts#L14). This is correct, you want both to be `false` here. The default options tell Denali to apply injections to the object you are registering. Setting both to false tells Denali to skip the injection step, which is what we want here since the raw Knex object we are registering will never have any injections directly on it.

File: tsconfig.json [hunk 4]
```diff
@@ -14,5 +14,5 @@
       "skipLibCheck": true,
       "noUnusedLocals": true
   },
-  "exclude": [ "blueprints/*/files/**" ]
+  "exclude": [ "blueprints/*/files/**", "dist", "tmp"]
```
Comment (davewasmer) [inline tsconfig.json]: We should add these to the denali-typescript blueprint

**Comment by davewasmer** [comment]
I'm a little confused by what github is showing for this PR - most of the changes it's showing as being added here are already in master.

**Comment by seawatts** [comment]
Yeah, I think they got out of sync somehow. I'm going to close this as it is no longer useful.
